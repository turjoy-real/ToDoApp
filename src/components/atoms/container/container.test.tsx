import React from 'react';
import {render} from '@testing-library/react-native';
import BasicContainer from './container';
import {Text} from 'react-native';
import {it} from '@jest/globals';

const MockedChild = () => <Text>Test Children</Text>;
describe('BasicContainer component', () => {
  it('renders children correctly', () => {
    const {getByText} = render(
      <BasicContainer>
        <MockedChild />
      </BasicContainer>,
    );
    expect(getByText('Test Children')).toBeTruthy();
  });

  it('matches snapshot', () => {
    const {toJSON} = render(
      <BasicContainer>
        <MockedChild />
      </BasicContainer>,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
