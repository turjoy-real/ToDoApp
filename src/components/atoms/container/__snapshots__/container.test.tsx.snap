// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicContainer component matches snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#000",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <Text>
    Test Children
  </Text>
</View>
`;
